### ISSUE TITLE
`no-standalone-expect` not behaving correctly inside fixture

### ISSUE THREAD

**Issue by tomupson** [mskelton/eslint-plugin-playwright#400]
Hi,

I've noticed an issue where the `no-standalone-expect` rule is not behaving as expected. Specifically, this happens inside fixtures, with assertions performed after a call to `page.evaluate`. See the example below:

```typescript
import { test as base, expect } from '@playwright/test';

type ExampleFixtures = {
  clipboardContents: string;
};

export const test = base.extend<ExampleFixtures>({
  clipboardContents: async ({ page }, use) => {
    await expect(page).toHaveURL('example.com'); // Works fine
    const text = await page.evaluate(() => navigator.clipboard.readText());
    await expect(page).toHaveURL('example.com'); // Error: Expect must be inside of a test block
    await use(text);
  }
});

export { expect } from '@playwright/test';
```

<img width="755" height="113" alt="Image" src="https://github.com/user-attachments/assets/940c32c3-3efa-476f-8ba8-1abd899f4dde" />

On the `no-standalone-expect` rule details, I don't see any mention of fixtures, so if this rule is not expected to work inside of fixtures then I may just need to disable the rule. But the [official playwright documentation contains examples with `expect` used inside fixtures](https://playwright.dev/docs/test-fixtures#worker-scoped-fixtures), and my tests run as expected, so to my this feels like an error in the rule.

Any help would be much appreciated!
